modules/light-clients/08-wasm/types/wasm_test.go

```diff
@@ -0,0 +1,92 @@
+package types_test
+
+import "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+
+func (suite *TypesTestSuite) TestGetCodeHashes() {
+	testCases := []struct {
+		name      string
+		malleate  func()
+		expResult func(codeHashes types.CodeHashes)
+	}{
+		{
+			"success: empty code hashes",
+			func() {},
+			func(codeHashes types.CodeHashes) {
+				suite.Require().Empty(codeHashes.Hashes)
+			},
+		},
+		{
+			"success: non-empty code hashes",
+			func() {
+				types.AddCodeHash(suite.chainA.GetContext(), GetSimApp(suite.chainA).AppCodec(), []byte("codehash"))
+			},
+			func(codeHashes types.CodeHashes) {
+				suite.Require().Len(codeHashes.Hashes, 1)
+				suite.Require().Equal([]byte("codehash"), codeHashes.Hashes[0])
+			},
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+		suite.Run(tc.name, func() {
+			suite.SetupTest()
+			tc.malleate()
+
+			codeHashes := types.GetCodeHashes(suite.chainA.GetContext(), GetSimApp(suite.chainA).AppCodec())
+			tc.expResult(codeHashes)
+		})
+	}
+}
+
+func (suite *TypesTestSuite) TestAddCodeHash() {
+	suite.SetupTest()
```

Comment:
`suite.SetupTest` is automatically called at the beginning of each test suite test